src/utilities/helperFunctions.js

```diff
@@ -14,3 +13,51 @@ export const validate = (input, setErrorMessage) => {
 
   return validated;
 };
+
+export const toCapitalized = function capitalizeString(str) {
+  if (!str) return '';
+
+  const words = str.split(' ');
+  const capitalizedWords = words.map(
+    (word) =>
+      `${word.charAt(0).toUpperCase()}${
+        word.length > 1 ? word.substring(1) : ''
+      }`
+  );
+
+  return capitalizedWords.join(' ');
+};
+
+// returns a date in local time (GMT +/- local timezone)
+export const toLocalDate = function convertTimestampToLocalDate(timestamp) {
+  return new Date(timestamp * 1000);
+};
+
+// returns an hour string (12-hour format) in the city's local time
+export const getHourString = function getFormattedHourStringFromDate(
+  date,
+  timezone
+) {
+  const UTCTimeInHours = date.getUTCHours();
+  const secondsInHour = 3600;
+  const cityUTCOffsetInHours = timezone / secondsInHour;
+  // rounding to correct for unusual timezones, i.e. UTC +3:30
+  let cityTime = Math.round(UTCTimeInHours + cityUTCOffsetInHours);
+  if (cityTime < 0) cityTime += 24;
+  else if (cityTime > 23) cityTime %= 24;
+  let hourString;
+
+  if (cityTime < 12) {
+    if (cityTime === 0) hourString = 'midnight';
+    else hourString = `${cityTime} am`;
+  } else if (cityTime > 11) {
+    if (cityTime === 12) hourString = 'noon';
+    else hourString = `${cityTime - 12} pm`;
+  }
+
+  return hourString;
+};
+
+export const getIconSrc = function getWeatherIconSrcURL(iconId) {
```

Comment:
Never thought about splitting this to a separate function, definitely have to refactor my portion to use this! Great idea!